.changeset/pretty-eagles-relate.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage/cli': patch
+'@backstage/plugin-scaffolder-backend': patch
```

Comment:
Let's make a different changeset for scaffolder-backend saying bump esbuild.

Reply:
done, thx!